wlutil/launch.py

```diff
@@ -0,0 +1,96 @@
+import logging
+from .wlutil import *
+
+# The amount of memory to use when launching
+launch_mem = "16384"
```

Comment:
This smells a little. Is this exposed somewhere?

Reply:
This just matches the default firesim config. It's not exposed right now. I think stuff like this needs to go into a "board" abstraction for FireMarshal. I also think that the 'launch' capability may merge with FireSim at some point which would also require some new abstractions (a virtual agfi or something).